### ISSUE TITLE
Community Moderation - no self-moderation

### ISSUE THREAD

**Issue by andymarden** [online-go/online-go.com#2468]
Currently you get to vote in a game where you were the reporter. Suggest fixing it so you don't get to moderate on games where you are involved.

**Comment by benjaminpjones** [comment]
I agree that you shouldn't get the report if you are the accused.  OTOH it isn't so bad if you put in one vote towards reports where you were the reporter.  If there is bias or misreporting, there are two other voters who can veto.

**Comment by GreenAsJade** [comment]
Just confirming that its a deliberate choice that you can vote on a report you raised, but an oversight that you can vote on a report complaining about you.
